server/routes/auth.ts

```diff
@@ -320,15 +320,18 @@ authRoutes.post('/jellyfin', async (req, res, next) => {
       // with admin permission
       settings.main.mediaServerType = MediaServerType.JELLYFIN;
       user = new User({
-        email: body.email,
+        email: body.email || account.User.Name,
         jellyfinUsername: account.User.Name,
         jellyfinUserId: account.User.Id,
         jellyfinDeviceId: deviceId,
         jellyfinAuthToken: account.AccessToken,
         permissions: Permission.ADMIN,
         avatar: account.User.PrimaryImageTag
           ? `${jellyfinHost}/Users/${account.User.Id}/Images/Primary/?tag=${account.User.PrimaryImageTag}&quality=90`
-          : gravatarUrl(body.email ?? '', { default: 'mm', size: 200 }),
+          : gravatarUrl(body.email || account.User.Name, {
```

Comment:
Why was this added? Afaik gravatar only uses emails

Reply:
Yep, but the `gravatar-url` package throws an error if the provided string is empty, but send a default profile picture when the provided a string/email doesn't have a corresponding profile picture.
I could replace the `|| account.User.Name` by something like `|| 'none'` if you thing it makes more sense.

Reply:
Email will always be there though. And if it's not there an empty string would give the default image (atleast it used to. Does it not now?)

Reply:
Email should be here yes. I added this because during one of my test I came across an empty email and `gravatar-url` throwed an error because of it.
See https://github.com/sindresorhus/gravatar-url/blob/main/index.js#L4-L6

Reply:
Ah makes sense. I assumed since overseerr was also just passing in email only